templates/rhsm-katello-reconfigure.erb

```diff
@@ -36,7 +50,8 @@ then
     --server.prefix="$PREFIX" \
     --server.port="$PORT" \
     --rhsm.repo_ca_cert="%(ca_cert_dir)s$KATELLO_SERVER_CA_CERT" \
-    --rhsm.baseurl="$BASEURL"
+    --rhsm.baseurl="$BASEURL" \
+    $REPOMD_GPG_PARAM
```

Comment:
This can be empty in which case the backslash might do something unexpected. The same applies below